pkgs/development/python-modules/pylev/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "pylev";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
```

Comment:
Why not `fetchPypi`?

Reply:
The pypi tarball for this and others don't include tests. Some of them also don't include a `setup.py` and so need to be built with poetry, so once we bootstrap poetry we actually will be making most of these packages build from GitHub so they can run with tests.

Reply:
I think I forgot to instruct that for `pylev` and `pastel` there should be a comment like `No tests in PyPi Tarball`.